sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1204 @@
+  intel/llvm#13257
+- Moved `slm_allocator` ESIMD APIs into `experimental` namespace. intel/llvm#13901
+- Removed deprecated `usm_system_allocator` aspect. intel/llvm#13279
+- Removed `get_child_group` API from experimental
+  [`sycl_ext_oneapi_root_group`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_root_group.asciidoc)
+  extension. intel/llvm#13482
+- Simplified template arguments related to `simd_view` of many ESIMD APIs. intel/llvm#13231
+- Removed ESIMD `atomic_op::predec`. intel/llvm#14480
+- Dropped interfaces from revision 1 of experimental
+  `sycl_ext_oneapi_group_sort` extension. intel/llvm14531
+- Changed return type of `command_graph::begin_recording` and
+  `command_graph::end_recording` from `void` to `bool` in the experimental
+  `sycl_ext_oneapi_graph` extension. intel/llvm#13480
+
+Breaking changes were also made to compiler flags:
+
+- Removed `-fsycl-link-huge-device-code` flag (it was deprecated in favor of
+  `-flink-huge-device-code`). intel/llvm#14731
+- Updated `-sycl-std` to disallow selection of SYCL 1.2.1. intel/llvm#14544
+- Removed deprecated `-fsycl-[add|link]-targets` flag. intel/llvm#13834
+- Removed deprecated `-foffload-static-lib` and `-foffload-whole-static-lib`.
+  Corresponding functionality is already available without the need to pass any
+  special options. intel/llvm#13835
+- Deprecated `-fsycl-disable-range-rounding` flag in favor of the new
+  `-fsycl-range-rounding`. intel/llvm#12715
+
+## Known Issues
+
+- On Windows, the Unified Runtime's Level Zero leak check does not work correctly with
+  the default contexts on Windows. This is because on Windows the release
+  of the plugin DLLs races against the release of static global variables
+  (like the default context).
+- Intel Graphic Compiler's Vector Compute backend does not support O0 code and
+  often gets miscompiled, produces wrong answers and crashes. This issue directly
+  affects ESIMD code at O0. As a temporary workaround, we have optimize ESIMD code
+  even in O0 mode.
+  [00749b1e8](https://github.com/intel/llvm/commit/00749b1e8e3085acfdc63108f073a255842533e2)
+- [new] Use of some SYCL math built-ins (like `abs` or `clz`) in a program where
+  `sycl/ext/intel/esimd.hpp` header is included causes compilation errors.
+  This will be fixed in the next release (intel/llvm#14793)
+- [new] When using `sycl_ext_oneapi_matrix` extension it is important for some
+  devices to use the sm version (Compute Capability) corresponding to the device
+  that will run the program, i.e. use `-fsycl-targets=nvidia_gpu_sm_xx` during
+  compilation. This particularly affects matrix operations using `half` data
+  type. For more information on this issue consult with
+  https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#wmma-restrictions
+
+commit https://github.com/intel/llvm/commit/c30769b122d99eb4d05bcb78f15e593491fe31ae
```

Comment:
Not clear to me what  the list of UR commits from here to the end of the notes for, a sentence or two of context before it would be useful.